Material.Blazor/Foundation/BatchingJsRuntime.cs

```diff
@@ -74,27 +85,19 @@ private async void Timer_Elapsed(object sender, ElapsedEventArgs e)
             }
         }
 
-        /// <summary>
-        /// Same as <see cref="JSRuntimeExtensions.InvokeVoidAsync(IJSRuntime, string, object[])"/>, except calls are batched in 20ms intervals.
-        /// </summary>
-        /// <param name="identifier"></param>
-        /// <param name="args"></param>
-        /// <returns></returns>
+
+        /// <inheritdoc/>
         public async Task InvokeVoidAsync(string identifier, params object[] args)
         {
             var call = new Call(identifier, args);
             queuedCalls.Enqueue(call);
             timer.Start();
             await call.Task;
         }
-        /// <summary>
-        /// Same as <see cref="JSRuntimeExtensions.InvokeAsync{TValue}(IJSRuntime, string, object?[]?)"/>
-        /// </summary>
-        /// <typeparam name="T"></typeparam>
-        /// <param name="identifier"></param>
-        /// <param name="args"></param>
-        /// <returns></returns>
-        public async Task<T> InvokeAsync<T>(string identifier, params object[] args)
+
+
+        /// <inheritdoc/>
+        public async ValueTask<T> InvokeAsync<T>(string identifier, params object[] args)
```

Comment:
I'm contemplating a PR that would also support batching for `InvokeAsync<T>`, but that requires a `Task<T>` return type. Any reason this must be a `ValueTask<T>`?

Reply:
Not my strong point and I'm happy to follow your lead.

Reply:
Changed back to `Task<T>` in 9fd9b54